vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -406,7 +391,7 @@ def update_field_visibility(options={})
     update_field_read_only(options)
   end
 
-  def update_field_visibility_linked_clone(options={}, f)
+  def update_field_visibility_linked_clone(_options = {}, f)
```

Comment:
@gmcculloug I think we still need that in case only one parameter is passed in, right?